docs/notebooks/DESC_Solve_from_pyQSC.ipynb

```diff
@@ -0,0 +1,250 @@
+      "text/plain": [
+       "<Figure size 864x864 with 2 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plot_comparison(\n",
+    "    eqs=[desc_eq, eq_fit],\n",
+    "    labels=[\"DESC-solved equilibrium\", \"NAE surfaces\"],\n",
+    "    zeta=2,\n",
+    "    figsize=(12, 12),\n",
+    "    theta=0,\n",
```

Comment:
What does the plot look like if you include the theta* contours? That might answer my previous question about if lambda is supposed to be zero or not

Reply:
![image](https://user-images.githubusercontent.com/22567458/224153186-84f91e7c-6a21-4f6e-a353-0b3923321ceb.png)
Looks pretty similar on the symmetry planes but a bit off on the others. Lambda probably isn't zero but I'm not sure if it's possible/easy to compute what it should be.